Vic2ToHoI4/Source/Hoi4Outputter/outDifficultySettings.cpp

```diff
@@ -0,0 +1,27 @@
+#include "../Configuration.h"
+#include "../HOI4World/HoI4Country.h"
+#include <fstream>
+
+void outputDifficultySettings(
+	const std::vector<std::shared_ptr<HoI4::Country>>& greatPowers,
+	const Configuration& theConfiguration
+) {
+	std::string diffSet = "";
+	for (const auto& GP: greatPowers)
```

Comment:
Instead of generating a string that you'll use just below, move this loop to the output section below and directly output the lines there,